src/pyhf/workspace.py

```diff
@@ -325,15 +325,15 @@ def get_measurement(self, **config_kwargs):
 
     def _get_measurement(self, **config_kwargs):
         """See `Workspace::get_measurement`."""
-        poi_name = config_kwargs.get('poi_name')
+        poi_name = config_kwargs.pop('poi_name', None)
```

Comment:
Maybe I'm missing something (@kratsg?) but as no check is performed anywhere here (in `workspace.py`) what is the advantage of switching from `get` to `pop` here?

Reply:
well, the advantage is we know when the configuration parameter is used. `config_kwargs` will be modified by deleting the `poi_name` key, so we can check if there's extra keys used... such as perhaps a typo.